src/runner/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
-name = "fluvio-runner-local"
-version = "0.4.0"
+name = "fluvio-run"
+version = "0.7.4"
```

Comment:
Tying this to fluvio version means, we have to update them every time.  We should just change this to basic version `0.1.0`

Reply:
Hmm, gotcha. In that case I wonder if we should not even touch the Cargo.toml version for this crate at all, just use VERSION in our deploy scripts for pushing this to packages.fluvio.io. Because when users install via `install.sh`, we need a way to make sure they get a matching `fluvio` and `fluvio-run`.

Reply:
Set to `0.1.0` in Cargo.toml 👍

Reply:
Yup.  This is where need to override for publishing